internal/expr/expr_test.go

```diff
@@ -519,7 +577,7 @@ comment */ WHERE x = $Address.&d`,
 		err:   `cannot parse expression: column 22: missing closing parentheses`,
 	}, {
 		query: "SELECT (name, id) WHERE id = $Person.*",
-		err:   `cannot parse expression: column 30: asterisk not allowed in input expression "$Person.*"`,
+		err:   `cannot parse expression: column 30: invalid asterisk input placement "$Person.*"`,
```

Comment:
Now that I re-read it I still love it but I am not sure whether `asterisk input` is a well defined thing or whether we should go for `invalid asterisk placement` or `invalid asterisk placement in input`. Thoughts?

Reply:
Changed to the second of the two suggestions, that seems better.